packages/api-server/src/convert-tx.ts

```diff
@@ -347,3 +408,46 @@ function publicKeyToEthAddress(publicKey: HexString): HexString {
       .toString("hex");
   return ethAddress;
 }
+
+async function cacheAutoCreateAccount(
+  polyjuiceRawTx: HexString,
+  ethTxHash: Hash,
+  fromEthAddress: HexString,
+  rpc: GodwokenClient,
+  gasLimit: HexNumber,
+  gasPrice: HexNumber,
+  value: HexNumber
+): Promise<[string, string] | undefined> {
+  const registryAddress: EthRegistryAddress = new EthRegistryAddress(
```

Comment:
how about re-using `verifyEnoughBalance` function here from validator.ts?

Reply:
For different error messge,
`from id not found by from Address: ${fromEthAddress}, have you deposited?`
OR
`insufficient balance, require ${requireBalance.toString()}, got ${balance}`

Reply:
Updated